### ISSUE TITLE
Cannot open Sales Invoice list view

### ISSUE THREAD

**Issue by rtdany10** [resilient-tech/india-compliance#500]
### Information about bug

![image](https://user-images.githubusercontent.com/52111700/223951238-0f44c3f7-23d3-4c3a-8192-78328b3e7948.png)

### Version

India Compliance: v14.2.1 (HEAD)
ERPNext: v14.18.1 (HEAD)
Frappe Framework: v14.27.0 (HEAD)

### Relevant log output / Stack trace / Full Error Message.

```shell
Uncaught (in promise) TypeError: ic.is_e_invoice_enabled is not a function
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sagarvora** [comment]
I think this might be a duplicate of https://github.com/resilient-tech/india-compliance/issues/203

Can you share the ouptut of following Client Script for List:
```
console.log(ic, window.ic);
```

**Comment by sagarvora** [comment]
Closing as duplicate of #203, will reopen that one.

**Comment by sagarvora** [comment]
The fix has been release in v14.3.0, please try it out.
